From c3b464c15666b835a6277d8de7cff145807391da Mon Sep 17 00:00:00 2001
From: krish <krish@cloudera.com>
Date: Fri, 19 Dec 2014 17:38:14 -0800
Subject: [PATCH 0190/1173] [useradmin] Disable password update for LDAP users

---
 apps/useradmin/src/useradmin/forms.py |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/apps/useradmin/src/useradmin/forms.py b/apps/useradmin/src/useradmin/forms.py
index 5cc395b..f091a20 100644
--- a/apps/useradmin/src/useradmin/forms.py
+++ b/apps/useradmin/src/useradmin/forms.py
@@ -96,6 +96,11 @@ class UserChangeForm(django.contrib.auth.forms.UserChangeForm):
     if self.instance.id:
       self.fields['username'].widget.attrs['readonly'] = True
 
+    if desktop_conf.AUTH.BACKEND.get() == 'desktop.auth.backend.LdapBackend':
+      self.fields['password1'].widget.attrs['readonly'] = True
+      self.fields['password2'].widget.attrs['readonly'] = True
+      self.fields['password_old'].widget.attrs['readonly'] = True
+
   def clean_password(self):
     return self.cleaned_data["password"]
 
-- 
1.7.9.5

