From f66d2ab3e3e884013d00631dafc2e3eac890aa92 Mon Sep 17 00:00:00 2001
From: Enrico Berti <hello@enricoberti.com>
Date: Thu, 27 Nov 2014 19:42:06 +0100
Subject: [PATCH 0326/1173] [oozie] Removed id from widgets

---
 .../oozie/templates/editor/workflow_editor.mako    |    2 --
 1 file changed, 2 deletions(-)

diff --git a/apps/oozie/src/oozie/templates/editor/workflow_editor.mako b/apps/oozie/src/oozie/templates/editor/workflow_editor.mako
index ba0be53..9f001eb 100644
--- a/apps/oozie/src/oozie/templates/editor/workflow_editor.mako
+++ b/apps/oozie/src/oozie/templates/editor/workflow_editor.mako
@@ -387,8 +387,6 @@ ${ commonheader(_("Workflow Editor"), "Oozie", user) | n,unicode }
         <span data-bind="editable: name, editableOptions: {enabled: $root.isEditing(), placement: 'right'}"></span>
       <!-- /ko -->
 
-      &nbsp;&nbsp;&nbsp;<span style="font-weight: normal; font-style: italic" data-bind="text: id"></span>
-
       <!-- ko if: widgetType() == 'decision-widget' -->
         <div class="inline pull-right" data-bind="visible: $root.isEditing() && $root.workflow.getNodeById(id()) && $root.workflow.getNodeById(id()).children().length <= 1 && ! oozieExpanded()">
           <a href="javascript:void(0)" data-bind="click: $root.removeWidget"><i class="fa fa-times"></i></a>
-- 
1.7.9.5

