From b7211b19d246729c5ca8113225506583205e7a4e Mon Sep 17 00:00:00 2001
From: Erick Tryzelaar <erickt@cloudera.com>
Date: Fri, 7 Nov 2014 12:11:33 -0800
Subject: [PATCH 0610/1173] [spark] Make sparker-shell cwd independent

---
 apps/spark/sparker-shell |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/apps/spark/sparker-shell b/apps/spark/sparker-shell
index c8a0ca1..575fc0a 100755
--- a/apps/spark/sparker-shell
+++ b/apps/spark/sparker-shell
@@ -1,3 +1,5 @@
 #!/bin/bash
 
+cd `dirname $0`
+
 exec java -cp "java/sparker-repl/target/lib/*:java/sparker-repl/target/sparker-repl-3.7.0-SNAPSHOT.jar" com.cloudera.hue.sparker.repl.Main -usejavacp "$@"
-- 
1.7.9.5

