From 2e550fbdbfed9bd05efd9c87a1a3247ede357592 Mon Sep 17 00:00:00 2001
From: Erick Tryzelaar <erickt@cloudera.com>
Date: Fri, 27 Feb 2015 10:21:45 -0800
Subject: [PATCH 0952/1173] [spark] Fix the Makefile to remove a circular
 dependency

---
 apps/spark/Makefile |   24 ++++++++++--------------
 1 file changed, 10 insertions(+), 14 deletions(-)

diff --git a/apps/spark/Makefile b/apps/spark/Makefile
index 9902a72..efbe16b 100644
--- a/apps/spark/Makefile
+++ b/apps/spark/Makefile
@@ -28,29 +28,25 @@ JAVA_LIB = $(SPARK_ROOT)/java-lib
 
 LIVY_DIR = $(SPARK_ROOT)/java
 LIVY_TARGET_DIR = $(LIVY_DIR)/livy-assembly/target/scala-2.10
-LIVY_ASSEMBLY = livy-assembly-$(MAVEN_VERSION).jar
-LIVY = $(JAVA_LIB)/$(LIVY_ASSEMBLY)
+LIVY_ASSEMBLY_JAR = livy-assembly-$(MAVEN_VERSION).jar
+LIVY_JAR = $(JAVA_LIB)/$(LIVY_ASSEMBLY)
 
 HAVE_JAVA_SRC = $(wildcard $(LIVY_DIR))
 
-compile: $(LIVY)
+compile: $(LIVY_JAR)
 
-ifneq (,$(HAVE_JAVA_SRC))
+ifneq (,$(wildcard $(LIVY_DIR)))
+$(LIVY_JAR): $(shell find $(LIVY_DIR) -type f)
+	mkdir -p $(JAVA_LIB)
+	@echo "--- Building Desktop spark"
+	cd $(LIVY_DIR) && mvn clean install -DskipTests $(MAVEN_OPTIONS)
+	cp $(LIVY_TARGET_DIR)/$(LIVY_ASSEMBLY_JAR) $(LIVY_JAR)
 
 clean::
 	rm -Rf $(JAVA_LIB)
 	cd $(LIVY_DIR) && mvn clean $(MAVEN_OPTIONS) || :
-
-$(LIVY_TARGET_DIR)/$(LIVY_ASSEMBLY): $(shell find $(LIVY_DIR) -type f)
-	@echo "--- Building Desktop spark"
-	cd $(LIVY_DIR) && mvn clean install -DskipTests $(MAVEN_OPTIONS)
-
-$(LIVY): $(LIVY_TARGET_DIR)/$(LIVY_ASSEMBLY)
-	mkdir -p $(JAVA_LIB)
-	cp $(LIVY_TARGET_DIR)/$(LIVY_ASSEMBLY) $(LIVY)
-
 else
-$(LIVY):
+$(LIVY_JAR):
 	$(error Cannot build spark jars without source)
 endif
 
-- 
1.7.9.5

