From 27e3bfe78a1f60f2a2517dd3447ec049e67b88eb Mon Sep 17 00:00:00 2001
From: Romain Rigaux <romain@cloudera.com>
Date: Thu, 12 Mar 2015 20:28:23 -0700
Subject: [PATCH 1069/1173] [doc] Help command to fix setup for tarball
 install

---
 dist/README |    1 +
 1 file changed, 1 insertion(+)

diff --git a/dist/README b/dist/README
index 54c7555..8817bc1 100644
--- a/dist/README
+++ b/dist/README
@@ -16,6 +16,7 @@ PREFIX=/usr/share make install
 
 ## If you don't have the permissions, you will need to 'sudo' the command and
 ## make sure that the 'hue' user has write access to the 'logs' directory and 'desktop/desktop.db'.
+## e.g. sudo chown -R hue:hue /usr/local/hue
 
 ## Run!
 ${PREFIX}/hue/build/env/bin/supervisor
-- 
1.7.9.5

