From f8f6e15c7df49407131894ad6955bc3563034b24 Mon Sep 17 00:00:00 2001
From: Erick Tryzelaar <erickt@cloudera.com>
Date: Mon, 16 Mar 2015 11:08:16 -0700
Subject: [PATCH 1090/1173] [livy] Ignore the dependency-reduced-pom.xml

---
 .gitignore |    1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index f00f9d7..856b265 100644
--- a/.gitignore
+++ b/.gitignore
@@ -77,6 +77,7 @@ target
 
 # Maven's dependency reduced pom
 desktop/libs/hadoop/java/dependency-reduced-pom.xml
+apps/spark/java/livy-core/dependency-reduced-pom.xml
 
 # Reviewboard config file
 .reviewboardrc
-- 
1.7.9.5

