From 5e90e1cc7944f7ee67e3e1dc67a2f97de2ba50ad Mon Sep 17 00:00:00 2001
From: Romain Rigaux <romain@cloudera.com>
Date: Thu, 19 Mar 2015 16:54:39 -0700
Subject: [PATCH 1132/1173] [oozie] Warning about using HBase credentials

---
 .../oozie/templates/editor2/common_workflow.mako   |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/apps/oozie/src/oozie/templates/editor2/common_workflow.mako b/apps/oozie/src/oozie/templates/editor2/common_workflow.mako
index 711f0fa..f01d9e4 100644
--- a/apps/oozie/src/oozie/templates/editor2/common_workflow.mako
+++ b/apps/oozie/src/oozie/templates/editor2/common_workflow.mako
@@ -499,6 +499,10 @@
 <script type="text/html" id="common-action-credentials">
   <em data-bind="visible: $root.credentials() == null || $root.credentials().length == 0">${ _('No available credentials.') }</em>
   <select data-bind="visible: $root.credentials() != null && $root.credentials().length > 0, options: $root.credentials, selectedOptions: properties.credentials" size="5" multiple="true"></select>
+
+  <em data-bind="visible: properties.credentials.indexOf('hbase') != -1">
+    ${ _('Requires hbase-site.xml in job path') }
+  </em>
 </script>
 
 
-- 
1.7.9.5

